web/webpack.prod.config.js

```diff
@@ -4,6 +4,11 @@ const commonConfig = require('./webpack.common.config.js');
 
 module.exports = merge(commonConfig, {
   plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify('production'),
```

Comment:
It's a bit of a convention in node-land (not standardized, just yet) to specify the environment through `NODE_ENV`. I grexplain how React uses it [here](https://github.com/mozilla/voice-web/pull/616#issuecomment-342828596).

Re stringifying, I'm just gonna quote [the webpack docs](https://webpack.js.org/plugins/define-plugin/):
> Note that because the plugin does a direct text replacement, the value given to it must include actual quotes inside of the string itself. Typically, this is done either with either alternate quotes, such as '"production"', or by using JSON.stringify('production').